templates/Shop/Homepage/Block/_carousel.html.twig

```diff
@@ -0,0 +1,21 @@
+<h2 class="ui center aligned huge header">{{ 'sylius.homepage.brace_yoursel_winter_is_coming'|trans }}</h2>
```

Comment:
I'm afraid we should take a text from block here  according to our DoD definition